### ISSUE TITLE
[Webpack WARNING] critical dependency

### ISSUE THREAD

**Issue by gongzza** [kaorun343/vue-property-decorator#32]
### Environment
OS: CentOS 7.3
node: v6.10.3
npm: 3.10.10
vue-property-decorator: 5.2.0
webpack: 3.3.0

#### Test
karma: 1.7.0
phantomjs: 2.1.14

when i test, i got this message:
```
WARNING in ./node_modules/vue-property-decorator/lib/vue-property-decorator.js
4:24-31 Critical dependency: require function is used in a way in which dependencies cannot be statically extracted
webpack: Compiled with warnings.
25 07 2017 14:07:43.501:INFO [karma]: Karma v1.7.0 server started at http://0.0.0.0:9876/
25 07 2017 14:07:43.507:INFO [launcher]: Launching browser PhantomJS with unlimited concurrency
25 07 2017 14:07:43.522:INFO [launcher]: Starting browser PhantomJS
25 07 2017 14:07:44.415:INFO [PhantomJS 2.1.1 (Linux 0.0.0)]: Connected on socket 3Lwn5pn5_bq63C9PAAAA with id 42841957
PhantomJS 2.1.1 (Linux 0.0.0) ERROR
  Error: Cannot find module "."
  at index.ts:15327
```
windows and macos passed.. 
my repo is [this](https://github.com/gongzza/vue-typescript-starter.git)

**Comment by kaorun343** [comment]
@gongzza 

Hi.

I'll fix this issue soon by updating build settings.

**Comment by fqybzhangji** [comment]
I have encountered this problem, has not yet resolved

**Comment by fqybzhangji** [comment]
@kaorun343 
I have encountered this problem, has not yet resolved

**Comment by kaorun343** [comment]
Could you try v5.2.1?

**Comment by fqybzhangji** [comment]
@kaorun343  
Thinks,The problem is solved and  v5.2.1 is OK
